requirements.txt

```diff
@@ -0,0 +1,5 @@
+scipy>=0.13
+numpy>=1.8
+netCDF4>=1.0.6
+matplotlib>=1.3.1
+pandas>=0.15.1
```

Comment:
Is there a reason why pandas>=0.15.1?

Reply:
I just copied the versions from the `setup.py` file.

Reply:
I questioned this regarding [James' comment](https://github.com/pacificclimate/osprey/pull/31#pullrequestreview-485413088). But if that's what the original RVIC requires, I believe that will be fine.